### PR TITLE
strapi-provider-email-mailgun: updates documentation

### PR DESCRIPTION
**Pull request by meck93**
Signed-off-by: Moritz Eck <moritz.eck@gmail.com>

### What does it do?
- updates the documentation

### Why is it needed?
- Saves users with a European mailgun host some time during the setup and the integration. 

### Related issue(s)/PR(s)
- None, I noticed it when I tried to use the mailgun email provider.

### CODE REVIEW

File: packages/strapi-provider-email-mailgun/README.md [hunk 1]
```diff
@@ -45,7 +45,11 @@ module.exports = ({ env }) => ({
     provider: 'mailgun',
     providerOptions: {
       apiKey: env('MAILGUN_API_KEY'),
-      domain: env('MAILGUN_DOMAIN'), //Required if you have an account with multiple domains
+      // Required if you have an account with multiple domains
```
Comment (derrickmehaffy) [inline packages/strapi-provider-email-mailgun/README.md:48]: Please keep comments in line with the config option

File: packages/strapi-provider-email-mailgun/README.md [hunk 2]
```diff
@@ -45,7 +45,11 @@ module.exports = ({ env }) => ({
...
+      domain: env('MAILGUN_DOMAIN'),
```
Comment (derrickmehaffy) [inline packages/strapi-provider-email-mailgun/README.md:49]: Please state this is optional and try to shorten the comment to a single line with the config option. Default should be `api.us.mailgun.net`

**Review by derrickmehaffy — CHANGES_REQUESTED** [review]
We will need this changes made here soon as we are moving our documentation to a new repo. If you are unable to make the changes in the next few hours, I will close this and make the changes on our new repo here: https://github.com/strapi/documentation

**Comment by meck93** [comment]
Hi @derrickmehaffy, thanks for the review. I have updated the documentation according to your feedback.

File: packages/strapi-provider-email-mailgun/README.md [hunk 3]
```diff
@@ -46,6 +46,7 @@ module.exports = ({ env }) => ({
     providerOptions: {
       apiKey: env('MAILGUN_API_KEY'),
       domain: env('MAILGUN_DOMAIN'), //Required if you have an account with multiple domains
+      host: 'api.us.mailgun.net', //Optional. If domain region is Europe: 'api.eu.mailgun.net'
```
Comment (derrickmehaffy) [inline packages/strapi-provider-email-mailgun/README.md:49]: It should be `host: env('MAILGUN_HOST', 'api.us.mailgun.net')`

**Review by derrickmehaffy — CHANGES_REQUESTED** [review]

**Review by derrickmehaffy — APPROVED** [review]
LGTM

**Comment by derrickmehaffy** [comment]
Pending tests then good to merge

**Pull request merged**